platform/features/conductor/src/ConductorCapabilityDecorator.js

```diff
@@ -0,0 +1,64 @@
+/*****************************************************************************
+ * Open MCT Web, Copyright (c) 2014-2015, United States Government
+ * as represented by the Administrator of the National Aeronautics and Space
+ * Administration. All rights reserved.
+ *
+ * Open MCT Web is licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ *
+ * Open MCT Web includes source code licensed under additional open source
+ * licenses. See the Open Source Licenses file (LICENSES.md) included with
+ * this source code distribution or the Licensing information page available
+ * at runtime from the About dialog for additional information.
+ *****************************************************************************/
+/*global define*/
+
+define(
+    ['./ConductorTelemetryCapability'],
+    function (ConductorTelemetryCapability) {
+        'use strict';
+
+        /**
+         * Decorates the `capabilityService` such that any exposed `telemetry`
+         * capabilities have their requests mediated by the time conductor.
+         *
+         * @constructor
+         * @memberof platform/features/conductor
+         * @implements {CapabilityService}
+         * @param {platform/features/conductor.ConductorService} conductorServe
+         *        the service which exposes the global time conductor
+         * @param {CapabilityService} capabilityService the decorated service
+         */
+        function ConductorCapabilityDecorator(conductorService, capabilityService) {
```

Comment:
So, it's the telemetry _capability_ that gets decorated, by way of the `capabilityService`

That said, it probably would be simpler to decorate the `telemetryService` directly and skip capabilities altogether. Byproduct of doing it this way would be that:

1. Domain objects which explicitly override their `telemetry` capability to not use the `telemetryService` will not use the time conductor; we don't have anything like this, so I don't think this is good or bad
2. We might end up going back to a capability-based approach if we need to scope the applicability of the time conductor. We've heard vague uses for wanting to scope this but don't have anything concrete for how this would look.

Based on (2) I slightly prefer the capability-based approach, but I'm fine with taking the simpler service-based approach, since those use cases aren't clear enough to say that capability decoration would really facilitate them anyway.